lib/membrane_ffmpeg_swscale/swscaler.ex

```diff
@@ -0,0 +1,242 @@
+defmodule Membrane.FFmpeg.SWScaler do
```

Comment:
This name is shorter, but a bit confusing, as the package name is *_swscale, and this is swscale**R**. Also, it does more than the scaling now. I'd suggest `SWScale.Converter` like we have in [swresample](https://hexdocs.pm/membrane_ffmpeg_swresample_plugin/Membrane.FFmpeg.SWResample.Converter.html)